events/api.py

```diff
@@ -1146,6 +1147,13 @@ def update(self, instance, validated_data):
 
     def to_representation(self, obj):
         ret = super(EventSerializer, self).to_representation(obj)
+
+        request = self.context['request']
```

Comment:
I think `if self.context['request'].accepted_...` would be enough since you don't use this variable anywhere else.